app/controllers/users_controller.rb

```diff
@@ -1,3 +1,27 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user= User.new
+  end
+
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
thêm code xử lý trường hợp user bị nil nè 
`return if @user` 
`flash[:warming] = "user not found"`
`redirect_to root_path`